examples/pipeline-stack/index.ts

```diff
@@ -10,7 +10,7 @@ import * as ssp from '../../lib'
 import * as team from '../teams'
 
 export default class PipelineStack extends cdk.Stack {
-    constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    constructor(scope: cdk.App, id: string, _props?: cdk.StackProps) {
```

Comment:
whats up with _props?

Reply:
Linting and IDE complains that props are never used. But it is important to keep them in the signature. So _ before property name is a common convention for optional properties and at lease VSCode automatically understands it and does not highlight this as an issue. I was thinking may be as a next step could fix linting to drop the warning for such cases as well.